packages/cojson/src/coValues/coPlainText.ts

```diff
@@ -0,0 +1,136 @@
+import { CoValueCore } from "../coValueCore.js";
+import { JsonObject } from "../jsonValue.js";
+import {
+  DeletionOpPayload,
+  InsertionOpPayload,
+  OpID,
+  RawCoList,
+} from "./coList.js";
+
+export type StringifiedOpID = string & { __stringifiedOpID: true };
+
+export function stringifyOpID(opID: OpID): StringifiedOpID {
+  return `${opID.sessionID}:${opID.txIndex}:${opID.changeIdx}` as StringifiedOpID;
+}
+
+type PlaintextIdxMapping = {
+  opIDbeforeIdx: OpID[];
+  opIDafterIdx: OpID[];
+  idxAfterOpID: { [opID: StringifiedOpID]: number };
+  idxBeforeOpID: { [opID: StringifiedOpID]: number };
+};
+
+const segmenter = new Intl.Segmenter("en", { granularity: "grapheme" });
```

Comment:
[The baseline for this API](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Segmenter) is only 2024 and it isn't supported by React Native.

We either include a polyfill or fallback in case the API is missing.

Also it would be batter if we move this evaluation inside `RawCoPlainText`, so the polyfill/fallback will be required only when using `RawCoPlainText`